services/QuillLMS/client/app/bundles/Teacher/components/classrooms/classroom_student_section.tsx

```diff
@@ -489,6 +489,15 @@ const ClassroomStudentSection = ({
     )
   }
 
+  const logInMethod = ({ email, provider }) => {
+    if (provider === 'Google') { return 'Google or email' }
+    if (provider === 'Clever') { return 'Clever' }
+    if (provider === 'Canvas') { return 'Canvas' }
```

Comment:
Nbd but we might want to constantize the provider names since we're using them for logic now.

Reply:
Good point